README.md

```diff
@@ -10,51 +10,36 @@ We recommend including the .d.ts file by downloading this package through [npm](
 ```bash
 
 # Install types and save in package.json as a development dependency
-npm install @microsoft/microsoft-graph-types --save-dev
+npm install @microsoft/microsoft-graph-types-beta --save-dev
 
 ```
 
-
-![GIF showing intellisense and autocompletion for Microsoft Graph entities in Visual Studio Code ](https://github.com/microsoftgraph/msgraph-typescript-typings/raw/master/typings-demo.gif)
+![GIF showing intellisense and autocompletion for Microsoft Graph entities in Visual Studio Code ](https://github.com/microsoftgraph/msgraph-beta-typescript-typings/blob/main/typings-demo.gif)
```

Comment:
I think we will need to record an alternative gif for this. This is referencing v1.0 types.

Reply:
@nikithauc  I have SnagIt on my computer so I could help with this capture in case you don't have a a screen capture app.

Reply:
I have removed these lines now. Will create a follow up task to update this. The gifs in both the repos should be updated.

Reply:
https://github.com/microsoftgraph/msgraph-beta-typescript-typings/issues/6